commons/generator/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "@devsoutinho/generator",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "gen": "cross-env NODE_OPTIONS='--import tsx' plop --plopfile=plopfile.ts",
```

Se você estiver utilizando windows é importante que o arquivo tenha aspas `"` ao invés de `'`:

```suggestion
    "gen": "cross-env NODE_OPTIONS="--import tsx" plop --plopfile=plopfile.ts",
```